test/Common/DateTimeExtensionsTests.cs

```diff
@@ -0,0 +1,62 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FluentAssertions;
+using System;
+
+using System.Drawing;
+
+namespace MatrTech.Utilities.Extensions.Common.UnitTests
+{
+    [TestClass]
+    public class DateTimeExtensionsTests
+    {
+        [TestMethod]
+        [DataRow(DayOfWeek.Monday, 7)]
+        [DataRow(DayOfWeek.Tuesday, 1)]
+        [DataRow(DayOfWeek.Wednesday, 2)]
+        [DataRow(DayOfWeek.Thursday, 3)]
+        [DataRow(DayOfWeek.Friday, 4)]
+        [DataRow(DayOfWeek.Saturday, 5)]
+        [DataRow(DayOfWeek.Sunday, 6)]
+        public void NextDayOfWeek_SourceAsMonday_GetsNextDayOfWeek(DayOfWeek dayOfWeek, int offset)
```

Comment:
SourceAsMonday klopt denk ik niet meer, EachDayOfWeek is het geworden zo te zien.